js/widgets/widget.theme.js

```diff
@@ -29,17 +29,30 @@
 
 $.mobile.widget.theme = {
 	_create: function() {
+		var that = this;
 		this._super();
 		if ( !this.options.enhanced && this.options.theme ) {
```

Comment:
Actually, in a simple setup, they're visible:

```HTML
<!doctype html>
<html>
<head>
	<title>RequireJS template</title>
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<link rel="stylesheet" href="css/themes/default/jquery.mobile.css"/>
	<script src="external/requirejs/require.js"></script>
	<script src="js/requirejs.config.js"></script>
	<script>
	requirejs.config( { baseUrl: "js" } );
	require( [ "widgets/popup" ], function() {
		$( function() {
			$( "#the-popup" ).popup();
			$( "#open-the-popup" ).on( "click", function() {
				$( "#the-popup" ).popup( "open" );
			} );
		} );
	} );
	</script>
</head>
<body>
	<a id="open-the-popup" href="#">Open</a>
	<div id="the-popup" data-history="false">
		<p>This is a popup</p>
	</div>
</body>
</html>
```

If you put the above in a file in the repo root and open it, you will see the popup briefly appear before it's torn out of the DOM.